modules/mobile/app/models/mobile/v0/templates/community_care_appointment.rb

```diff
@@ -41,26 +41,11 @@ def location
             },
             lat: nil,
             long: nil,
-            phone: phone_captures,
```

Comment:
This was done in error. The phone number is the patient's, not the facility's. The patient's phone number is also captured in these responses as `patient_phone_number` but that is simply passed through and not parsed. I did some testing on staging and found that the front end does its own phone number validation. It only allows 10 digit numbers (typical US phone numbers) and returns them as numerical strings (i.e., with any dashes or other characters removed). The web app also displays the patient phone numbers as a plain numerical string, so the way we're handling it should be consistent.